test/e2e-test/flareact-function.test.ts

```diff
@@ -0,0 +1,94 @@
+import util = require('node:util');
```

Comment:
We need to merge this file with flareact.test.js to avoid code duplication and to avoid run the same test twice.

Reply:
Actually this comment applies to whole file not to this specific line :)